src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIPipelineService.java

```diff
@@ -128,4 +156,52 @@ private String localizedParamOptionLabel(Locale locale, String workflowName, Str
             return paramName + "." + optionName;
         }
     }
+
+    private List<SavedPipelineParameters> getSavedPipelineParameters(IridaWorkflow workflow, Locale locale) {
```

Comment:
Add javadoc

Reply:
Updated in ff30fb64d75064b84c44c6d33bf4579f0f827d7e